docs/code-of-conduct.rst

```diff
@@ -0,0 +1,33 @@
+Code of Conduct
+===============
+
+Write the Docs is a global community of documentarians who share information, discuss ideas, and work together to improve the art and science of documentation.
+
+While diversity is one of our strengths, it can lead to communication issues. To that end, we have a few ground rules that we ask people to adhere to when they're participating within this community and its projects. These rules apply equally to founders, mentors, admins, and those seeking conversation, discussion, help, and guidance -- in short, to all participants.
+
+This isn't an exhaustive list of things that you must do, or can't do. Rather, take it in the spirit in which it's intended.It's a guide to make it easier to enrich all of us -- the technical community and the conferences, meetups, and other forums we hope to guide new participants to.
+
+This code of conduct applies to all spaces associated with the Write the Docs community, whether virtual or face-to-face. This includes Slack, the discussion forum, social media, conferences, meetups, and anywhere else Write the Docs might show up. In addition, violations of this code outside these spaces may affect a person's ability to participate within them. 
+
+* **Be friendly and patient.**
+
+* **Be welcoming.** We strive to be a community that welcomes and supports people of all backgrounds and identities. This includes, but is not limited to, members of any race, ethnicity, culture, national origin, colour, immigration status, social and economic class, educational level, sex, sexual orientation, gender identity and expression, age, size, family status, political belief, religion, and mental and physical ability.
+
+* **Be considerate.** Your words and your work will be used by other people, and you in turn will depend on the words and the work of others. Any decision you take will affect users and colleagues, and you should take those consequences into account when making decisions. Remember that we're a world-wide community, so you might not be communicating in someone else's primary language.
+
+* **Be respectful.** Not all of us will agree all the time, but disagreement is no excuse for poor behavior and poor manners. We might all experience some frustration now and then, but we cannot allow that frustration to turn into a personal attack. It’s important to remember that a community where people feel uncomfortable or threatened is not a productive one. Members of the Write the Docs community should be respectful when dealing with others inside and outside the Write the Docs community.
```

Comment:
Based on this discussion -- 

This was: "Members of the Write the Docs community should be respectful when dealing with other members as well as with people outside the Write the Docs community."